scripts/lint-packages.js

```diff
@@ -18,6 +18,10 @@ let path = require('path');
 let packages = glob.sync(path.dirname(__dirname) + '/packages/@{react,spectrum}-*/*/package.json');
 let errors = false;
 
+function ignore(jsonPackage) {
```

Comment:
Why was this updated? I would prefer not to have this change for the whole repo.

Reply:
To avoid linting the parcel-resolver-alias-fix repo. Instead it looks like I can just move it to `packages/dev` in order to avoid linting.